docs/Color-Picker/Color-Picker.md

```diff
@@ -1 +1,24 @@
-TODO
\ No newline at end of file
+## Requirements
+
+### Part 1: App-Only
+* The audience will listen to the orchestra play four pieces of music.
+* Each audience member will use the app to pick from six available colors (RGB, CMY).
```

Comment:
Need to follow-up with Dr. Steve Palmer in New Mexico for final color choices.

Reply:
To clarify: Audience members will only be able to pick a single color (or emotion) at a time, correct?